src/api.js

```diff
@@ -1,12 +1,17 @@
 import instance from '@redhat-cloud-services/frontend-components-utilities/files/interceptors';
+export { instance };
 
-export const getApplicationSchema = async (application, apiVersion = 'v1', resourceType = '') => {
-    return await instance.get(`/api/${application}/${apiVersion}/user-config/${resourceType}`);
+export const getApplicationSchema = async (application, apiVersion = 'v1', resourceType = '', url) => {
+    try {
+        return await instance.get(`/api/${application}/${apiVersion}${url || `/user-config/${resourceType}`}`);
+    } catch (e) {
+        console.log(e);
```

Comment:
Are you sure we want to catch errors without any notification?

Reply:
Yeah, this was only for testing purposes. I'll remove it